docs/reference/ownership-mechanism-and-readonly-flags.ipynb

```diff
@@ -0,0 +1,268 @@
+   "source": [
+    "To create a variable with sole ownership of a buffer, use the `copy()` method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab601216-6657-45b8-81cf-54d3f1e73e76",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var['x', 4:6].copy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8b00d395-8fa9-4f38-99a7-2572b3ae39d6",
+   "metadata": {},
+   "source": [
+    "By default, `copy()` returns a deep copy.\n",
+    "Shallow copies can be made by specifying `deep=False`, which preserves shared ownership of underlying buffers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f9c888bc-3439-4cd5-9120-28e20634a3e6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "shallow_copy = var['x', 4:6].copy(deep=False)\n",
+    "shallow_copy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "be0a620c-db87-4c06-967a-ff1ec260c567",
+   "metadata": {},
+   "source": [
+    "## Data arrays\n",
+    "\n",
+    "As a result of the sharing mechanism, extra care must now be taken in some cases, just like when working with any other Python library.\n",
+    "Consider the following example, using the same variable as data and as a coordinate:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "13230d53-de54-4ab0-9422-494141263c68",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = sc.DataArray(data=var, coords={'x': var})\n",
+    "da += 666 * sc.units.m\n",
+    "da"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2df62313-8ecc-49bf-874b-dbd40786015a",
+   "metadata": {},
+   "source": [
+    "The modification unintentionally also affected the coordinate.\n",
+    "However, if we think of data arrays and coordinate dicts as Python-like objects, then the behavior should not be surprising.\n",
+    "\n",
+    "Note that the original `var` is also affected:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f6997509-8f8c-4cb4-ab71-26ab671aa6b1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8e9387ab-75a9-4c8f-aa5e-63491f2efa41",
+   "metadata": {},
+   "source": [
+    "To avoid this, use `copy()`, e.g.,:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "50f43a3d-7f6d-4717-a0f5-5df5dbb5f719",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = sc.DataArray(data=var.copy(), coords={'x': var.copy()})\n",
+    "da += 666 * sc.units.m\n",
+    "da"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "98faae5d-4251-409f-87f7-9c5675ccdeb6",
+   "metadata": {},
+   "source": [
+    "Apart from the standard and pythonic behavior, one advantage of this is that creating data arrays from variables is typically cheap, without inflicting copies of potentially large objects.\n",
+    "\n",
+    "A related concept are read-only flags.\n",
```

Comment:
Please make this a sub(sub)section to make it easier to find if someone has questions.

Reply:
Extracted the whole paragraph into new section.